core/src/test/java/org/apache/calcite/test/SqlToRelConverterTest.java

```diff
@@ -3076,16 +3076,14 @@ void checkCorrelatedMapSubQuery(boolean expand) {
     sql(sql).ok();
   }
 
-  @Disabled("CALCITE-985")
   @Test void testMerge() {
-    final String sql = "merge into emp as target\n"
-        + "using (select * from emp where deptno = 30) as source\n"
-        + "on target.empno = source.empno\n"
-        + "when matched then\n"
-        + "  update set sal = sal + source.sal\n"
-        + "when not matched then\n"
-        + "  insert (empno, deptno, sal)\n"
-        + "  values (source.empno, source.deptno, source.sal)";
+    final String sql = "merge into empnullables e "
```

Comment:
Can you end each line of SQL with "\n" (and no space). It makes the SQL easier to read inside `SqlToRelConverterTest.xml`.

Reply:
Done